docs/content/advanced/backend/batch-jobs/customize-import.md

```diff
@@ -0,0 +1,109 @@
+
+Although this documentation specifically targets import strategies, you can use the same steps to overwrite any batch job strategy in Medusa, including export strategies.
+
+## Prerequisites
+
+### Medusa Components
+
+It's assumed that you already have a Medusa server installed and set up. If not, you can follow our [quickstart guide](../../../quickstart/quick-start.md) to get started.
+
+### Redis
+
+Redis is required for batch jobs to work. Make sure you [install Redis](../../../tutorial/0-set-up-your-development-environment.mdx#redis) and [configure it with your Medusa server](../../../usage/configurations.md#redis).
+
+### PostgreSQL
+
+If you use SQLite during your development, it’s highly recommended that you use PostgreSQL when working with batch jobs. Learn how to [install PostgreSQL](../../../tutorial/0-set-up-your-development-environment.mdx#postgresql) and [configure it with your Medusa server](../../../usage/configurations.md#postgresql-configurations).
+
+## Overwrite Batch Job Strategy
+
+The steps required for overwriting a batch job strategy are essentially the same steps required to create a batch job strategy with a minor difference. For that reason, this documentation does not cover the basics of a batch job strategy.
+
+If you’re interested to learn more about batch job strategies and how they work, please check out the [Create Batch Job Strategy documentation](./create.md).
+
+### 1. Create a File
+
+You must store batch job strategies in the `src/strategies` directory of your Medusa server. They are either TypeScript or JavaScript files.
+
+So, for example, you can create the file `src/strategies/import.ts`.
+
+### 2. Create a Class
+
+The batch job strategy class must extend the `AbstractBatchJobStrategy` class which you can import from Medusa’s core repository.
+
+For example, you can define the following class in the file you created:
+
+```tsx
+import { AbstractBatchJobStrategy, BatchJobService } from '@medusajs/medusa'
+import { EntityManager } from 'typeorm'
+
+class ImportStrategy extends AbstractBatchJobStrategy {
+  protected batchJobService_: BatchJobService
+  protected manager_: EntityManager
+  protected transactionManager_: EntityManager
+
+  processJob(batchJobId: string): Promise<void> {
+    throw new Error('Method not implemented.')
+  }
+  buildTemplate(): Promise<string> {
+    throw new Error('Method not implemented.')
+  }
+}
+
+export default ImportStrategy
+```
+
+### 3. Set the batchType Property
+
+Every batch job strategy class must have the static property `batchType` defined. It determines the type of batch job this strategy handles.
+
+Since only one batch job strategy can handle a batch job type, you can overwrite Medusa’s default batch job strategies by using the same `batchType` value in your custom strategy.
+
+So, for example, to overwrite the product import strategy set the `batchType` property in your strategy to `product-import`:
+
+```tsx
```

Comment:
nit: technically this isn't tsx but pure typescript

```suggestion
```typescript
```

Reply:
That's true, it's exported from Notion in that format. I added a vale rule to spot it in the future